src/openrct2-ui/windows/GameBottomToolbar.cpp

```diff
@@ -601,8 +601,9 @@ static void window_game_bottom_toolbar_draw_news_item(rct_drawpixelinfo* dpi, rc
                 return;
 
             auto clipCoords = ScreenCoordsXY{ 10, 19 };
-
-            if (peep->Is<Staff>() && peep->AssignedStaffType == StaffType::Entertainer)
+            auto* staff = peep->As<Staff>();
+            auto* guest = peep->As<Guest>();
```

Comment:
I'd move this statement 10 lines further, to be just before the `if` that needs it.

Reply:
I thought it made sense to have them together. I don't really mind where they go though